### PR TITLE
Revert "Subscription Caching Service (#87)"

### PR DESCRIPTION
**Pull request by ivan-stankov-golang**
This reverts commit c2a8501a818fc4dde1d0b61ffa4c70fb4239d08a.

### CODE REVIEW

File: config/test_config.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ func SetTestDefaultConfig() {
 		"BRUM_PERSISTANCE_DATABASE_STRATEGY": "all_in_one_db",
 		"BRUM_PERSISTANCE_TABLE_STRATEGY":    "all_in_one_table",
 		"BRUM_BACKUP_ENABLED":                "false",
-		"BRUM_BACKUP_DIRECTORY":              "/home/basicrum_backup",
+		"BRUM_BACKUP_DIRECTORY":              "/home/basicrum_archive",
```
Comment (ceckoslab) [inline config/test_config.go:22]: @ivan-stankov-golang is it possible that we revert more than needed here? 

This change seem to be unrelated. Could you check the whole PR for similar things that slipped?

**Comment by ivan-stankov-golang** [comment]
Closing this PR. Opened new PR: https://github.com/basicrum/front_basicrum_go/pull/93.
